.github/workflows/fixturegen.yml

```diff
@@ -22,13 +22,7 @@ jobs:
       - uses: actions/checkout@v1
       - name: Select Xcode
         run: sudo xcode-select -switch /Applications/Xcode_${{ matrix.xcode }}.app
-      - name: Build Package
-        working-directory: ./tools/fixturegen
-        run: swift build
       - name: Generate Fixture
-        working-directory: ./tools/fixturegen
-        run: swift run fixturegen --projects 1 --targets 1 --sources 1
-      - name: Build Tuist
-        run: swift build
+        run: ./fourier fixture --projects 1 --targets 1 --sources 1
```

Comment:
testing locally I think this may fail due to missing `--path` argument which appears to be required in Fourier whereas it is optional and a has a default `--path Fixture` when invoking `swift run fixturegen` directly.

```sh
$ ./fourier fixture --projects 1 --targets 1 --sources 1
No value provided for required options '--path'
```

Interestingly if I do specify it, it's relative to the fixturegen project, I wonder if there's a way to preserve the current working directory, but might be tricky when using `swift run` 🙈

```sh
$ ./fourier fixture --projects 1 --targets 1 --sources 1 MyFixture
$ git status
On branch tools-to-projects
Your branch is up to date with 'github/tools-to-projects'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        projects/fixturegen/MyFixture/
```

Reply:
I've fixed the Fourier command to default to `Fixture` relative from the current working directory.